code/modules/admin/topic.dm

```diff
@@ -2301,26 +2301,39 @@
 	else if(href_list["cryossd"])
 		if(!check_rights(R_ADMIN))
 			return
-		var/mob/living/carbon/human/H = locateUID(href_list["cryossd"])
-		if(!istype(H))
-			to_chat(usr, "<span class='warning'>This can only be used on instances of type /mob/living/carbon/human</span>")
+		var/mob/living/M= locateUID(href_list["cryossd"])
+		if(!ishuman(M) && !issilicon(M))
```

Comment:
You could maybe change the `ishuman()` here to be a variable, that way you don't have to call it again later on line 2317/2325.

Reply:
done!